### ISSUE TITLE
EquipmentOverlay: IOOBE: Index: -1

### ISSUE THREAD

**Issue by jani270** [NotEnoughUpdates/NotEnoughUpdates#1049]
```
java.lang.IndexOutOfBoundsException: Index: -1
	at java.util.Collections$EmptyList.get(Collections.java:4456)
	at io.github.moulberry.notenoughupdates.overlays.EquipmentOverlay.onClickItem(EquipmentOverlay.java:394)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_114_EquipmentOverlay_onClickItem_PlayerInteractEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at net.minecraftforge.event.ForgeEventFactory.onPlayerInteract(ForgeEventFactory.java:130)
```

**Comment by GrowlingGrizzly** [comment]
I think this is fixed in #1014

**Comment by jani270** [comment]
> I think this is fixed in #1014

You sure about that?
> `String itemType = Objects.requireNonNull(StringUtils.substringAfterLast(heldItemLore.get(heldItemLore.size() - 1), " "), "null");`

**Comment by GrowlingGrizzly** [comment]
https://github.com/NotEnoughUpdates/NotEnoughUpdates/pull/1014/commits/4a89351d926d842fbccd3a5671cc31ab0623cd70
